### PR TITLE
Makes Reishi and Ambrosia More Easily Obtainable

### PR DESCRIPTION
**Pull request by furrycactus**
Removes Reishi mycelium from the contraband list of MegaSeed Servitors and puts them in the regular stock list. Now they can be vended in hydroponics and purchased in the surface garden without machine hacking.

### CODE REVIEW

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -602,7 +603,7 @@
 		/obj/item/seeds/glowshroom = 3,
 		/obj/item/seeds/libertymycelium = 3,
 		/obj/item/seeds/nettleseed = 3,
```
Comment (Doxxmedearly) [inline code/game/machinery/vending_types.dm]:
```suggestion
		/obj/item/seeds/nettleseed = 3
```

**Comment by runecap** [comment]
You could probably make ambrosia more easily obtainable to. Strange only Hydroponics has access when they're not illegal anymore.

**Comment by skull132** [comment]
!review

**Comment by furrycactus** [comment]
> You could probably make ambrosia more easily obtainable to. Strange only Hydroponics has access when they're not illegal anymore.

Deus is premium stock, so you just need a coin for that, but I can put Vulgaris in the regular list.

**Pull request merged**